data/layout/telescope_positions-South-1LST.ecsv

```diff
@@ -1,42 +1,42 @@
-# %ECSV 0.9
+# %ECSV 1.0
 # ---
 # datatype:
-# - {name: telescope_name, datatype: string}
-# - {name: pos_x, unit: m, datatype: float64}
-# - {name: pos_y, unit: m, datatype: float64}
-# - {name: pos_z, unit: m, datatype: float64}
-# - {name: lat, unit: deg, datatype: float64}
-# - {name: lon, unit: deg, datatype: float64}
-# - {name: utm_east, unit: m, datatype: float64}
-# - {name: utm_north, unit: m, datatype: float64}
-# - {name: alt, unit: m, datatype: float64}
+# - {name: telescope_name, datatype: string, description: common telescope name}
+# - {name: pos_x, unit: m, datatype: float32, description: telescope position towards North relative to MST01 (central MST)}
+# - {name: pos_y, unit: m, datatype: float32, description: telescope position towards West relative to MST01 (central MST)}
+# - {name: pos_z, unit: m, datatype: float32, description: telescope position (centre of sphere) upwards relative to NNN}
 # meta:
+#   data_type: positionfile
+#   description: "telescope positions for CTA South (Paranal)"
 #   EPSG: 32719
 #   center_alt: !astropy.units.Quantity
 #     unit: &id001 !astropy.units.Unit {unit: m}
-#     value: 2147.0
+#     value: 2162.35
 #   center_easting: !astropy.units.Quantity
 #     unit: *id001
-#     value: 366822.0000008711
+#     value: 366822.017
 #   center_lat: !astropy.units.Quantity
 #     unit: &id002 !astropy.units.Unit {unit: deg}
-#     value: -24.68342915473787
+#     value: -24.683429
 #   center_lon: !astropy.units.Quantity
 #     unit: *id002
-#     value: -70.31634499364885
+#     value: -70.316345
 #   center_northing: !astropy.units.Quantity
 #     unit: *id001
-#     value: 7269465.999807887
+#     value: 7269466.999
```

Comment:
Suggest to have easting and northing togehter (not lat/lon inbetween)

Reply:
It is now alphabetically sorted. Do we want to lose that?

Reply:
I think it should be sorted by logic.

Reply:
I assume you mean the logic you used in the files in CTA gitlab and I changed here now to match that order